acme_diags/driver/enso_diags_driver.py

```diff
@@ -233,30 +243,99 @@ def run_diag(parameter):
+                         metrics_dict, ref_confidence_levels, test_confidence_levels,
+                         parameter)
                 utils.general.save_ncfiles(parameter.current_set,
                                            test_reg_coe, ref_reg_coe, diff, parameter)
     return parameter
+
+
+def run_diag_scatter(parameter):
+    variables = parameter.variables
+    run_type = parameter.run_type
+    # We will always use the same regions, so we don't do the following:
+    # x['region'] = parameter.nino_region
+    # y['region'] = parameter.regions[0]
+    x = {'var': 'TS', 'units': 'degC', 'region': 'NINO3'}
+    # todo: select same way nino region, average over the region selected and that becomes a point in the monthly
+    #  time series.
+    test_data = utils.dataset.Dataset(parameter, test=True)
+    ref_data = utils.dataset.Dataset(parameter, ref=True)
+    if parameter.print_statements:
+        print('run_type: {}'.format(run_type))
+    if run_type == 'model_vs_model':
+        x['test'] = calculate_nino_index_model(test_data, x['region'], parameter)
+        x['ref'] = calculate_nino_index_model(ref_data, x['region'], parameter)
+    elif run_type == 'model_vs_obs':
+        x['test'] = calculate_nino_index_model(test_data, x['region'], parameter)
+        x['ref'] = calculate_nino_index(x['region'], parameter)
+    else:
+        raise Exception('Invalid run_type={}'.format(run_type))
+    parameter.test_name_yrs = utils.general.get_name_and_yrs(parameter, test_data)
+    parameter.ref_name_yrs = utils.general.get_name_and_yrs(parameter, ref_data)
+    for y_var in variables:
+        if y_var == 'TAUX':
+            regions = ['NINO4']
+        else:
+            regions = ['NINO3']
+        for region in regions:
+            y = {'var': y_var, 'region': region}
+            test_data_ts = test_data.get_timeseries_variable(y_var)
+            ref_data_ts = ref_data.get_timeseries_variable(y_var)
+            y_region = default_regions.regions_specs[region]['domain']
+            test_data_ts_regional = test_data_ts(y_region)
+            ref_data_ts_regional = ref_data_ts(y_region)
+            # Domain average
+            test_avg = cdutil.averager(test_data_ts_regional, axis='xy')
+            ref_avg = cdutil.averager(ref_data_ts_regional, axis='xy')
+            # Get anomaly from annual cycle climatology
+            y['test'] = cdutil.ANNUALCYCLE.departures(test_avg)
+            y['ref'] = cdutil.ANNUALCYCLE.departures(ref_avg)
+            y['units'] = test_avg.units
+            if y_var == 'TAUX':
+                y['test'] *= 1000
+                y['ref'] *= 1000
+                y['units'] = '10^3 {}'.format(y['units'])
+            parameter.var_id = '{}-feedback'.format(y['var'])
+            title_tuple = (y['var'], y['region'], x['var'], x['region'])
+            parameter.main_title = 'Feedback, {} anomaly ({}) vs. {} anomaly ({})'.format(*title_tuple)
```

Comment:
Change `Feedback` to `Regression` to be main title and display in description in viewer

Reply:
On line 194, we set the main title to be "Regression coefficient" for the maps. Wouldn't it be slightly confusing to also have the scatterplots titled "Regression"? Then again, I suppose they are both displaying regressions.

Reply:
ah! I edited it moment ago. Removing 'Feedback, 'would be good.

Reply:
Ah ok sorry, sounds good

Reply:
@chengzhuzhang Is it ok to leave `-feedback` two lines above for `var_id`?

Reply:
Leaving `feedback` in `var_id` and `output_file`